testing/ui-tests/src/lib.rs

```diff
@@ -51,6 +52,12 @@ fn ui_tests() {
             .build(dispatch_errors::metadata_array_dispatch_error()),
     );
 
+    t.pass(
+        m.new_test_case()
+            .name("custom_values_foo")
+            .build(custom_values::metadata_custom_values_foo()),
+    );
```

Comment:
I'm happy with this for now! 

I'll raise an issue for a future PR to discuss how to test this more fully (eg here we've tested that the codegen doesn't fail in a fairly simple case, but it'd be good to also test that we can fetch custom values via the generated interface). It might be a bit fiddly because we'd want to do generate some metadata with custom values in and then use the subxt macro on it to generate the interface and allow us to test it. Will ponder :)

Reply:
I opened #1134, so after this merges we can add that :)